test/integration/sequelize.test.js

```diff
@@ -448,8 +450,10 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
             ].some(fragment => err.message.includes(fragment)));
           } else if (dialect === 'mssql') {
             expect(err.message).to.equal('Login failed for user \'bar\'.');
+          } else if (dialect === 'db2') {
+            expect(err.message).to.include('SQLSTATE=08004');
           } else {
-            expect(err.message.toString()).to.match(/.*Access denied.*/);
+            expect(err.message.toString()).to.match(/.*Access denied*./);
```

Comment:
Done by mistake?

Reply:
NOt sure if I did it as it is not specific to DB2 but reverting.